docs/user-event/install.mdx

```diff
@@ -26,6 +26,16 @@ yarn add --dev @testing-library/user-event
 </TabItem>
 </Tabs>
 
+Now simply import it in your tests:
+
+```js
+import userEvent from '@testing-library/user-event'
+
+// or
+
+const {default: userEvent} = require('@testing-library/user-event')
+```
```

Comment:
Maybe we should use `Tabs` here too. (Backslash to make the suggestion display.)
```suggestion
<Tabs defaultValue="esm" values={[
    { label: 'In EcmaScript Module', value: 'esm' },
    { label: 'In CommonJS', value: 'cjs' },
]}>
<TabItem value="esm">
  ```js
  import userEvent from '@testing-library/user-event'
  \```
</TabItem>
<TabItem value="cjs">
  ```js
  const {default: userEvent} = require('@testing-library/user-event')
  \```
</TabItem>
</Tabs>
```

Reply:
I was thinking it would be nice to show both variants by default in case users don't understand the difference between EcmaScript and CommonJS modules without seeing code. I also can't seem to get this to parse (event without the backslashes).